package.json

```diff
@@ -174,7 +174,8 @@
     "vite": "^3.2.4",
     "vite-plugin-html": "^3.2.0",
     "vite-plugin-pwa": "^0.13.3",
-    "vite-plugin-sentry": "^1.1.6",
+    "vite-plugin-sentry": "^1.1.7",
+    "vite-plugin-swc-react-refresh": "^2.2.1",
```

Comment:
**issue:** This seems wrong. Recently we updated swc plugin to 3.2.0 (`@vitejs/plugin-react-swc`), version 2.2.1 caused problems on intel platforms (#3205)